### PR TITLE
adding easyconfigs: CellRanger-ATAC-2.1.0.eb

### PR DESCRIPTION
**Pull request by VictorGoitea**

### CODE REVIEW

**Comment by verdurin** [comment]
@VictorGoitea welcome to EasyBuild, and thanks for your contribution.

The unit tests have picked up some problems in the PR, which I'll address in a review.

File: easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = "https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac"
+description = """Cell Ranger ATAC is a set of analysis pipelines that process 
+Chromium Single Cell ATAC data."""
+
+toolchain = SYSTEM
+
+# Download manually from https://support.10xgenomics.com/single-cell-atac/software/downloads/latest
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3f44ba66ae789c2ab03f9c380a6718ca']
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb:18]: @VictorGoitea apparently this is not a checksum in the required `SHA256` format.

File: easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+keepsymlinks = True
+
+sanity_check_paths = {
+    'files': ['bin/cellranger-atac', 'cellranger-atac'],
+    'dirs': ['bin/atac', 'lib'],
+}
+
+sanity_check_commands = ["cellranger-atac testrun --id=tiny"]
+
+moduleclass = 'bio'
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb:29]: Please also remove this newline - the unit tests are complaining.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb (hunk 1 above)
Comment (VictorGoitea) [reply easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb]: Hi @verdurin, fixed the checksum to sha256 format now. I am sorry about it, the company Cellranger provides instead md5sum in the downloading website.

File: easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb (hunk 2 above)
Comment (VictorGoitea) [reply easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb]: @verdurin I am not sure if I understand correctly which is the conflicting line, line 27 or 29? Is not 'bio' one of the defaults module classes?

Comment (verdurin) [reply easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb]: Thanks.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/24debd15054473fe30ae3298597ae95f for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @VictorGoitea!

**Pull request merged**